app/assets/stylesheets/reservations/_index.css.scss

```diff
@@ -0,0 +1,3 @@
+.count {
```

Comment:
This class is redundant – Bootstrap already provides a `.pull-right` utility class for this! :smiley: